src/reducers/kiosk.js

```diff
@@ -0,0 +1,24 @@
+import {combineReducers} from 'redux';
+
```

Comment:
This file has action constants, actions and reducers - more than the directory suggests.

Could be good to have different modules have similar things, e.g.

```
App/
  actionTypes.js
  actions.js
  App.js
  reducer.js
Foo
  Foo.js
  Foo.test.js
  Foo.css
  reducer.js
```

etc